beacon-chain/slasher/helpers.go

```diff
@@ -105,6 +106,15 @@ func validateAttestationIntegrity(att *ethpb.IndexedAttestation) bool {
 	return sourceEpoch < targetEpoch
 }
 
+// Validates the signed beacon block header integrity, ensuring we have no nil values.
+func validateBlockHeaderIntegrity(header *ethpb.SignedBeaconBlockHeader) bool {
+	// If a signed block header is malformed, we drop it.
+	if header == nil || header.Header == nil || len(header.Signature) < 96 || bytes.Equal(header.Signature, make([]byte, 96)) {
```

Comment:
This should always be 96 bytes, right?
```suggestion
	if header == nil || header.Header == nil || len(header.Signature) != 96 || bytes.Equal(header.Signature, make([]byte, 96)) {
```

Reply:
Use config value for this `params.BeaconConfig().BLSSignatureLength` ?

Reply:
Sorry I missed this in my review